cmd/ip/ip.go

```diff
@@ -0,0 +1,36 @@
+package ip
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+)
+
+var name, instance string
+
+// IPCmd manages reserved IPs
+var IPCmd = &cobra.Command{
+	Use:     "ip",
+	Aliases: []string{"ips"},
```

Comment:
We can have more aliases - {ip-address, ip-addresses, reserved-ip, reserved-ips}